src/service/extensions-impl.js

```diff
@@ -238,15 +241,19 @@ export class Extensions {
    * Returns the promise that will be resolved when the extension has been
    * loaded. If necessary, adds the extension script to the page.
    * @param {string} extensionId
+   * @param {string=} extensionVer
    * @param {boolean=} stubElement
    * @return {!Promise<!ExtensionDef>}
    */
-  loadExtension(extensionId, stubElement = true) {
+  loadExtension(extensionId,
+      extensionVer = DefaultExtensionVersion, stubElement = true) {
```

Comment:
we want to restrict on the the extensionVersion input. Can we add a check to make sure that extensionVersion is valid with a given extensionId before inserting script.